.buildkite/nightly.yml

```diff
@@ -44,3 +48,12 @@ steps:
     timeout_in_minutes: 60
     agents:
       system: x86_64-linux
+
+  - label: "Rebuild master branch"
+    trigger: "cardano-wallet"
+    branches: "master"
+    build:
+      message: "Nightly build with jormungandr integration tests"
```

Comment:
There's no `commands` here, is it triggering the normal `.buildkite/rebuild` somehow?

Reply:
Yes, it's meant to trigger the other pipeline. I just got it working. You can see what happens on Buildkite.

Reply:
This is what it does:
![Screenshot from 2020-04-23 12-11-15](https://user-images.githubusercontent.com/1019641/80051474-8ea64180-855b-11ea-8607-4676c7519d69.png)